console/console-init/ui/src/stories/DeviceList.stories.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { MemoryRouter } from "react-router";
+
+import { DeviceListAlert } from "modules/device";
+import { text } from "@storybook/addon-knobs";
+
+export default {
+  title: "Device"
+};
+
+export const deviceListAlert = () => (
```

Comment:
component name should be in Pascal case i.e. DeviceListAlert. If you are worries about same name as you have already your component. You can have other name i.e. DeviceAlert